azext_edge/tests/edge/support/create_bundle_int/helpers.py

```diff
@@ -262,9 +262,9 @@ def _get_namespace_determinating_files(
         level_1 = walk_result.pop(path.join(BASE_ZIP_PATH, clusterconfig_namespace))
         assert level_1["folders"] == ["clusterconfig"]
         assert not level_1["files"]
-        level_2 = walk_result.pop(path.join(BASE_ZIP_PATH, namespace, "clusterconfig"))
-        assert level_2["folders"] == ["billing"]
-        assert not level_2["files"]
+        # level_2 = walk_result.pop(path.join(BASE_ZIP_PATH, namespace, "clusterconfig"))
```

Comment:
What was the reason this is commented out ?

Reply:
I was a change I forgot to remove, will update!

Reply:
no more clusterconfig under aio namespace